xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/WCAGContext.java

```diff
@@ -269,6 +293,15 @@ protected Map<String, Integer> getViolationCountPerRule()
     {
         return violationCountPerRule;
     }
+    protected Map<String, Integer> getIncompleteCountPerRule()
```

Comment:
Missing line break before this line.
```suggestion

    protected Map<String, Integer> getIncompleteCountPerRule()
```